scala/private/phases/phase_dependency.bzl

```diff
@@ -64,3 +78,9 @@ def _get_unused_deps_mode(ctx):
         return ctx.attr.unused_dependency_checker_mode
     else:
         return ctx.toolchains["@io_bazel_rules_scala//scala:toolchain_type"].unused_dependency_checker_mode
+
+def _is_target_included(target, includes, excludes):
+    if len([exclude for exclude in excludes if target.startswith(exclude)]) > 0:
```

Comment:
Maybe instead of building list and checking length you could use loop with an if? For example
```
for exclude in excludes:
  if target.startswith(exclude):
    return False
```

Or maybe `if any([target.startswith(exclude) for exclude in excludes])` but I guess you would building new list each time however it might not be an issue.

Reply:
I'm leaning towards simple loop